sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -513,4 +553,18 @@ export interface RestoreKeyBackupOptions extends coreHttp.OperationOptions {}
 /**
  * An interface representing the options of the cryptography API methods, go to the {@link CryptographyClient} for more information.
  */
-export interface CryptographyOptions extends coreHttp.OperationOptions {}
+export interface CryptographyOptions extends coreHttp.OperationOptions {
```

Comment:
Same here.

Reply:
I moved this change out of this PR! Thanks!